aspnetcore/test/integration-tests.md

```diff
@@ -162,6 +162,9 @@ Create the `WebApplicationFactoryClientOptions` class and pass it to the <xref:M
 
 [!code-csharp[](~/../AspNetCore.Docs.Samples/test/integration-tests/IntegrationTestsSample/tests/RazorPagesProject.Tests/IntegrationTests/IndexPageTests.cs?name=snippet1)]
 
+> [!NOTE]
+> To avoid HTTPS redirection warnings in logs when using HTTPS Redirection Middleware, set `BaseAddress = new Uri("https://localhost")`
```

Comment:
```suggestion
***NOTE:*** To avoid HTTPS redirection warnings in logs when using HTTPS Redirection Middleware, set `BaseAddress = new Uri("https://localhost")`
```